Moderation-Policy.md

```diff
@@ -75,6 +76,26 @@ subject to Moderation.
 The Moderation Team is responsible for deciding what constitutes inappropriate
 behavior that may be subject to Moderation.
 
+## Outside Commentary via Social Media
+
+Comments by individuals made in any venue falling outside the explicit scope of
+responsibility of the Node.js Foundation, TSC, or Community Committee are
+explicitly not covered by this policy unless the individual is explicitly acting
+or commenting on behalf of the Node.js Foundation, TSC or Community Committee
+in an official capacity. The statements, views, and opinions contained in such
```

Comment:
isn't "in an official capacity" anywhere they've declared their affiliation?

In other words, If i stick "node.js TSC member" in a social media profile, isn't that entire profile and its contents acting and commenting on behalf of the foundation?

Reply:
I don't think so. I have that I work for Microsoft in my social media, but I don't act on behalf of them at all. I think the 'my tweets do not reflect my employers' bio stuff is just precautionary, because obviously your personal accounts don't have anything to do with the corporate entity you're a part of.

Reply:
Hmm - I guess it's surprising to me that social media posts that explicitly mention my affiliation would be exempt from the CoC.

If I wanted to, for example, ensure that my twitter comments were out of scope for Airbnb to be able to have comments on, I'd remove any mention of Airbnb from my profile. I'd expect the same for node affiliation.

Reply:
i expect a confusion to exist here as the twitter bio is mutable. for example: i do not list npm or node in my bio, however, because of the current situation, i added it, so people would know people to reach out to and why.

twitter does not track what your bio is at time of tweeting. therefore, in a situation where tweets are looked at, say 2-3 years later, there is no way to know what someone's twitter bio said at the time.

in general, i think this is a complicated and flimsy distinction. i totally empathy with the definition of this being complicated, but a twitter bio is not an appropriate way to tell, in my opinion.

Reply:
Fair point.

I'm concerned about the loophole for people to be able to violate the CoC in spirit by tweeting their abuse (as opposed to posting it on github), even if it doesn't violate the letter of the policy.

Reply:
"Official capacity" would be someone posting something to a Foundation owned account, like the `@nodejs` twitter handle, or someone explicitly trying to represent themselves as speaking on behalf of the foundation. For instance, if I went on twitter and said, "On behalf of the project..." that would be different than me saying on twitter, "I really think such and such sucks..."

That said, this is precisely why I add the second paragraph that says no matter what this exclusion says, your words are going to be interpreted as being associated with the project so please consider what you're going to say carefully.

This is also why this explicitly says such reports *may* be ignored rather than *will* or *shall*. Each will still have to be looked at and evaluated. Something that I say on Twitter is rightfully under more scrutiny than say almost anyone else involved in the project right now.

Reply:
As long as it still allows the TSC to *choose* to consider the report on a case-by-case basis, my concerns are satisfied for now (although removing human bias from that decision, towards enforcement, would be a nice change to make in the future).

Reply:
Same. I think as it is written it's good, really agree with @ljharb's statement above- just wanted to clarify about twitter bios.

Reply:
Yes, this *should* still leave room for the case by case determination. If the language can be strengthened there, please let me know.

Reply:
I agree it makes a reasonable balance here and I think we should handle any additional expectations for people in the org in a discussion like https://github.com/nodejs/TSC/issues/311

Reply:
Wouldn't adding an out here ("case-by-case") just be ripe for trolling, like the current situation (e.g. "it says in your policy you can do something about this person tweeting, why aren't you doing anything?", etc.)? IMHO just leaving someone's personal social media messages as their own no matter what, would help to avoid that kind of stuff better.

Reply:
I don't think it's practical to do that. If one is affiliated, and one makes a problematic statement *anywhere*, that can reflect poorly on the project. That doesn't mean there's an easy way to clearly limit it, nor that it's practical to do so, but the reality is that if you want complete freedom to say anything in any arena, then that's just not compatible with leadership in a project that wants to be inclusive.